event_battle_groups.txt

```diff
@@ -0,0 +1,83 @@
+Formations to avoid:
```

Comment:
is this file needed? Looks like it's the same data as enemy_battle_groups.py

Reply:
This file is not needed. I don't know how to remove it from the branch though. Do I just delete the file and then push a new commit?

Reply:
OK, I removed it.